packages/shadydom/src/utils.js

```diff
@@ -35,7 +35,7 @@ export const isTrackingLogicalChildNodes = (node) => {
 }
 
 export const isShadyRoot = (obj) => {
-  return Boolean(obj._localName === 'ShadyRoot');
+  return Boolean(obj instanceof DocumentFragment && obj._localName === 'ShadyRoot');
```

Comment:
Can we use `instanceof ShadowRoot` and eliminate the `_localName` check altogether?

Reply:
Yeah, I believe that should work now. Will try and see if that's green in testing.